docs/how-to-use.md

```diff
@@ -25,6 +26,23 @@ cd background-checks
 
 This starts up a new Docker Application with a stack of containers.
 
+## How to clean up docker containers
+
+* Stop all containers
+```
+docker kill $(docker ps -q) 
```

Comment:
```suggestion
docker compose stop
```

`docker kill $(docker ps -q)` stops all of the docker containers on the system, regardless of they are related to the background-checks application.